### DISCUSSION TITLE
Only plots with pool contract loads to farm

### DISCUSSION THREAD

**Discussion by xludodx** [Chia-Network/chia-blockchain#7348] [Support]
Hello,

I just installed the gui, created same wallets as for chia but only the plots newly created with a pool contract show up in plots. All others are showing faulty.
Any idea?
Thanks
Xludodx

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
